content/webapp/components/Tabs/Tabs.Switch.tsx

```diff
@@ -54,6 +55,8 @@ export type Props = {
   setSelectedTab: Dispatch<SetStateAction<string>>;
   isWhite?: boolean;
   trackWithSegment?: boolean;
+  isThin?: boolean;
+  selectedBorderColour?: PaletteColor;
```

Comment:
I'm curious about changes to existing components; maybe we can run through them?
If they are required, they should also be added to the `Tabs.Navigate` variant as they should be the same.